CONTRIBUTING.md

```diff
@@ -44,6 +44,28 @@ for example:
 git clone git@github.com:YOUR-USERNAME/narwhals.git
 ```
 
+### 3. Add the `upstream` remote
```

Comment:
like this we've got two `### 3.`, shall we update the numbers?

Reply:
done